server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/GetMessageListMethodTest.java

```diff
@@ -889,6 +898,30 @@ public void getMessageListShouldExcludeMessagesWhenInMailboxesFilterMatches() th
             .body(ARGUMENTS + ".messageIds", contains(message.getMessageId().serialize()));
     }
 
+    @Test
```

Comment:
Can we add it in junit group to run it on top of Cassandra-es?

Reply:
`@Category(BasicFeature.class)`?